### DISCUSSION TITLE
Is there a way to debug the output?

### DISCUSSION THREAD

**Discussion by pvazn** [openhtmltopdf/openhtmltopdf#61] [Q&A]
I am trying to place things using CSS but the output is not what I want or expect. In FF I can go to dev tools and see how the elements contribute to the structure, but FF's output does not match openhtmltopdf output even if I toggle "print media" on. What is your way of troubleshooting formatting issues?

**Comment by syjer** [comment]
hi @pvazn , the old school way before the dev tools existed: adding "border: 1px solid red" (and other colors) to the various elements.

**Reply by pvazn** [reply]
Thank you, that helped
